server/protocols/webadmin/webadmin-rabbitmq/src/test/java/org/apache/james/webadmin/routes/RabbitMQRoutesTest.java

```diff
@@ -110,4 +112,30 @@ void triggeringARebuildShouldCreateATask() {
             .statusCode(HttpStatus.OK_200)
             .body("", hasSize(1));
     }
+
+    @Test
+    void triggeringARebuildWhenTheQueueHasNotBeenInitializedShouldFail() {
+        when(mailQueueFactory.getQueue(any())).thenReturn(Optional.empty());
+        given()
+            .queryParam("task", "rebuild")
+            .queryParam("olderThan", "1d")
+        .when()
+            .post(RabbitMQRoutes.BASE_URL + "/spooler")
+        .then()
+            .statusCode(HttpStatus.INTERNAL_SERVER_ERROR_500)
+            .body("message", containsString("Spooler MailQueue not created"));
+    }
+
+    @Test
+    void triggeringARebuildWhenTheQueueHasNotTheRightTypeShouldFail() {
```

Comment:
With my advices on typing (using RabbitMQMailQueueFactory) this test is meaningless.